frappe/doctypes.py

```diff
@@ -0,0 +1,53 @@
+from typing import Dict
+from frappe.model.document import Document
+from frappe.model.base_document import get_controller
+
+import frappe
+
+FRAPPE_KEY = bytes("frappe", "utf8")
+
+
+def modulize(dt: str) -> str:
+	"""Convert string into importable Python namespace
+	"""
+	return dt.replace(" ", "").replace("-", "_")
+
+
+def generate_doctype_map() -> Dict[str, Document]:
```

Comment:
```suggestion

def generate_doctype_map() -> Dict[str, Type[Document]]:
```
The values are classes, not instances of those classes so it should be `Type` of `Document` (or its subtypes).

---
In general, what is the usecase/benefit of doing this? 🤔

Reply:
Updated the description about the why